### PR TITLE
Feature 2 : View Average Grade - Kimberly Roque

### PR DESCRIPTION
**Pull request by billy2anne**
Screen Shot 2020-05-08 at 5 19 00 PM

### CODE REVIEW

File: features/components/grade-table.js [hunk 1]
```diff
@@ -4,15 +4,18 @@ class GradeTable {
   }
 
   updateGrades(grades){
+    //emptyBody
     var tBody = this.tableElement.querySelector('tbody');
     $('tBody').empty();
+    var avgGrades = [];
```
Comment (WhiskeyJack96) [inline features/components/grade-table.js:10]: this should probably just be called "grades" right? its not a list of averageGrades yet

File: features/components/grade-table.js [hunk 2]
```diff
@@ -21,9 +24,20 @@ class GradeTable {
...
+
+    //Calculate Average Students'Grade
+    var sum = 0;
+    for( var i = 0; i < avgGrades.length; i++){
+      var studentGrade = avgGrades[i];
+      sum = sum + studentGrade;
+      var numberOfStudents = avgGrades.length;
+      var average = sum/numberOfStudents
     }
+    return average;
```
Comment (WhiskeyJack96) [inline features/components/grade-table.js:41]: No required change, 

You could also only calculate the sum in the for loop and then `return sum/grades.length`

File: features/components/grade-table.js (hunk 1 above)
Comment (billy2anne) [reply features/components/grade-table.js]: done with corrections.

File: features/components/grade-table.js (hunk 2 above)
Comment (billy2anne) [reply features/components/grade-table.js:40]: thanks for the feedback

**Pull request merged**